### ISSUE TITLE
boxdraw(?) glyphs cannot be rendered outside of a terminal

### ISSUE THREAD

**Issue by wael444** [ryanoasis/nerd-fonts#844]
#### 🗹 Requirements

- [X] I have searched the [issues](https://github.com/ryanoasis/nerd-fonts/issues) for my issue and found nothing related and/or helpful
- [X] I have searched the [FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting) for help
- [X] I have searched the [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki) for help

#### 🎯 Subject of the issue

Experienced behavior:
Box drawing glyphs outside of a terminal that renders the glyphs by themselves gets rendered

Expected behavior:
No glyphs/missing glyph character

Example symbols:
https://en.wikipedia.org/wiki/Box-drawing_character

#### 🔧 Your Setup

- Font: `JetBrainsMono Nerd Font Mono`
   - `/home/wael/.local/share/fonts/JetBrains Mono Regular Nerd Font Complete Mono.ttf`
   - Previously got it from `nerd-fonts-ttf` package, switched to github for testing.
- `st`, `xterm`, `alacritty`
- `Void 5.18.2_1 x86_64 GenuineIntel uptodate FFFFFFFFFFFFFF`

#### ★ Screenshots (Optional)
using `ImageMagick` and [stw](https://github.com/sineemore/stw)

ImageMagick (JetBrains-Mono-Regular)
![image](https://user-images.githubusercontent.com/47404953/174073101-d4e6e8bd-82a2-4d79-a422-9f556a75b3b4.png)
ImageMagick (JJetBrains-Mono-Regular-Nerd-Font-Complete)
![image](https://user-images.githubusercontent.com/47404953/174073019-b4b599f5-dc59-451c-a55e-a10ad751b2dd.png)

stw (JetBrains Mono:size=10)
![image](https://user-images.githubusercontent.com/47404953/174073190-d9c9dd9e-c38b-450a-8585-84f1b5b775c3.png)
stw (JetBrainsMono Nerd Font Mono:size=10)
![image](https://user-images.githubusercontent.com/47404953/174073232-e0546e38-dd78-453e-b3ec-55fc44ce542a.png)

commands used: 
```
stw -F 'JetBrainsMono Nerd Font Mono:size=10' -f white -b black pstree -U
```
```
stw -F 'JetBrains Mono:size=10' -f white -b black pstree -U
```
---
`@unreal` leading to the command output of `pstree -U` to a file called `unreal` in the same directory the command is being executed.
it can be substituted for `"$(pstree -U)"`
```
convert -size 1920x1080 -background '#1d2021' -fill '#ebdbb2' -font "JetBrains-Mono-Regular-Nerd-Font-Complete" -pointsize 15 label:@unreal <image output>
```
```
convert -size 1920x1080 -background '#1d2021' -fill '#ebdbb2' -font "JetBrains-Mono-Regular" -pointsize 15 label:@unreal <image output>
```

**Comment by Finii** [comment]
Hmm, works on my machine...

![image](https://user-images.githubusercontent.com/16012374/174575182-3429f239-59c7-443b-8983-2a6662c5e549.png)

* clone and compile `stw`
* download and install JetBrainsMono Ligatures Regular Complete from [here](https://github.com/ryanoasis/nerd-fonts/blob/master/patched-fonts/JetBrainsMono/Ligatures/Regular/complete/JetBrains%20Mono%20Regular%20Nerd%20Font%20Complete%20Mono.ttf)
* run stw, see screenshot

**Comment by Finii** [comment]
The Box Drawing Glyphs are at `2500` and following, these codepoints are not touched by `font-patcher`.

Maybe they were missing in an old JetBrains Mono version (that is the base for the patched font you use)?

You can check the font you have with `fontforge`, just scroll down to the `2500` codepoints...

![image](https://user-images.githubusercontent.com/16012374/174577190-eb617c5c-a475-4758-a95c-c8f6dc925069.png)

**Comment by wael444** [comment]
yes, it seems that now the boxdraw characters are rendered, thanks  
i think the issue now is my distributions `nerd-fonts`.

**Comment by wael444** [comment]
why doesnt the full font pack from nerd fonts work? https://github.com/ryanoasis/nerd-fonts/releases/download/v2.1.0/JetBrainsMono.zip

this still has issues.

**Comment by Finii** [comment]
This is an Issue with upstream (i.e. JetBrains)...

Nerd Fonts v2.1.0 used JetBrains Mono `1.0.2; ttfautohint (v1.8.3)`
Nerd Fonts v2.2.0 uses JetBrains Mono `2.251; ttfautohint (v1.8.3)`

The 1.0.2 has no boxdraw stuff, while the 2.251 has.
Find out at https://github.com/JetBrains/JetBrainsMono when they introduced it ;-)

**Comment by Finii** [comment]
https://github.com/JetBrains/JetBrainsMono/issues/84

**Comment by wael444** [comment]
where is Nerd Fonts v2.2.0 ?
